xendit-java-lib/src/main/java/com/xendit/model/DisbursementPH.java

```diff
@@ -0,0 +1,169 @@
+import com.xendit.Xendit;
+import com.xendit.exception.XenditException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.*;
+
+@Getter
+@Setter
+@Builder
+public class DisbursementPH {
+  @SerializedName("id")
+  private String id;
+
+  @SerializedName("reference_id")
+  private String referenceId;
+
+  @SerializedName("currency")
+  private String currency;
+
+  @SerializedName("amount")
+  private Integer amount;
+
+  @SerializedName("channel_code")
+  private String channelCode;
+
+  @SerializedName("description")
+  private String description;
+
+  @SerializedName("status")
+  private String status;
+
+  @SerializedName("created")
+  private Date created;
+
+  @SerializedName("updated")
+  private Date updated;
+
+  // optionals
+  @SerializedName("email_to")
+  private String[] emailTo;
+
+  @SerializedName("email_cc")
+  private String[] emailCC;
+
+  @SerializedName("email_bcc")
+  private String[] emailBcc;
+
+  private static DisbursementClient disbursementClient;
+
+  /**
+   * Get object disbursementPH by id
+   *
+   * @param headers
+   * @param id
+   * @return
+   * @throws XenditException
+   */
+  public static DisbursementPH getById(Map<String, String> headers, String id)
+      throws XenditException {
+    DisbursementClient client = getClient();
+    return client.getPHById(headers, id);
+  }
+
+  /**
+   * Get object disbursementPH by id
+   *
+   * @param id
+   * @return
+   * @throws XenditException
+   */
+  public static DisbursementPH getById(String id) throws XenditException {
+    return getById(new HashMap<>(), id);
+  }
+
+  /**
+   * Its create a client for DisbursementPH
+   *
+   * @return DisbursementClient
+   */
+  private static DisbursementClient getClient() {
+    if (isApiKeyExist()) {
+      if (disbursementClient == null
+          || !disbursementClient.getOpt().getApiKey().trim().equals(Xendit.apiKey.trim())) {
+        return disbursementClient =
+            new DisbursementClient(Xendit.Opt.setApiKey(Xendit.apiKey), Xendit.getRequestClient());
+      }
+    } else {
+      if (disbursementClient == null
+          || !disbursementClient
+              .getOpt()
+              .getApiKey()
+              .trim()
+              .equals(Xendit.Opt.getApiKey().trim())) {
+        return disbursementClient = new DisbursementClient(Xendit.Opt, Xendit.getRequestClient());
+      }
+    }
+    return disbursementClient;
+  }
+
+  /**
+   * check if api-key is exist or not
+   *
+   * @return boolean
+   */
+  private static boolean isApiKeyExist() {
+    return Xendit.apiKey != null;
+  }
+
+  /**
+   * Create v1 disbursement with required parameters
+   *
+   * @param xendit_idempotency_key
+   * @param reference_id
+   * @param currency
+   * @param channel_code
+   * @param account_name
+   * @param account_number
+   * @param description
+   * @param amount
+   * @return Disbursement
+   * @throws XenditException
+   */
+  public static DisbursementPH create(
```

Comment:
you need to provide another overload for this for each optional field.

Reply:
13b2edf36536b75f0da2e8fe89a3e5aabbd257d9